airgun/widgets.py

```diff
@@ -703,26 +728,16 @@ class SatFlashMessages(FlashMessages):
         //div[@class="toast-notifications-list-pf"]
 
     """
-
-    @property
-    def messages(self):
-        result = []
-        try:
-            for flash_div in self.browser.elements(
-                    './div[contains(@class, "alert")]', parent=self,
-                    check_visibility=True):
-                result.append(
-                    SatFlashMessage(self, flash_div, logger=self.logger))
-        except NoSuchElementException:
-            pass
-        return result
+    ROOT = '//div[@class="toast-notifications-list-pf"]'
+    MSG_LOCATOR = f'{ROOT}/div[contains(@class, "alert")]'
+    msg_class = SatFlashMessage
```

Comment:
Is it work here to call the class `SatFlashMessage` without `()` ?

Reply:
Yes, this is reference to class, not an instance.

It's instantiated in [`messages()` method](https://github.com/RedHatQE/widgetastic.patternfly/blob/master/src/widgetastic_patternfly/__init__.py#L2890)